css/style.css

```diff
@@ -1039,22 +1041,1370 @@ li { list-style-type: none; }
   position: relative;
   width: 1049px;
   height: 592px;
-  right: 72px;
+  right: 102px;
   top: 168px;
-  margin: 0 0 253.56px 0;
+  margin: 2px 0 0 10px;
   padding: 0;
   background: url(https://i.ibb.co/rZbd7qZ/Car-frame-group.png) no-repeat;
-  background-position: center;
+  background-size: 100%;
 }
 
 .image-car__webHero {
   display: flex;
   width: 93.13px;
   height: 37.77px;
   padding: 0;
-  border: 1px solid blue;
   background: url(https://i.ibb.co/QMG0bnM/Group-2222.png;) no-repeat;
   background-position: center;
   background-size: 100%;
-  margin: 0 18.64px 12.44px 0;
+  margin: 0 0 12.52px 818px;
+}
+
+@media screen and (max-width: 768px) {
+  .container {
+    display: flex;
+    flex-flow: column nowrap;
+    max-width: 768px;
+    height: 100%;
+    align-content: center;
+    justify-content: flex-start;
+    margin: 0 auto;
+  }
+
+  .menu__nav {
+    display: none;
+  }
+
+  .container__menu {
+    display: flex;
+    width: 719.68px;
+    height: 51px;
+    justify-content: space-between;
+    margin: 24px 33px 80.32px 15.33px;
+  }
+
+  .menu__logo {
+    display: flex;
+    width: 166px;
+    height: 51px;
+    background: url(https://i.ibb.co/x1r6ZBx/Tesla-Wordmark-Red.png;);
+    background-size: 100%;
+    background-position: left bottom 4px;
+    padding: 0;
+    margin: 0;
+  }
+
+  .menu__burger {
+    display: flex;
+    width: 35.23px;
+    height: 32px;
+    background: url(https://i.ibb.co/gzf1wRT/burger-ipad.png) no-repeat;
+    background-size: 100% 75%;
+    background-position: center center;
+    margin: 0 0 4px 0;
+    padding: 0;
+    border: 0;
+    outline: none;
+    cursor: pointer;
+  }
+
+  .container__sections {
+    display: flex;
+    flex-flow: column wrap;
+    max-width: 768px;
+    height: 100%;
+    justify-content: flex-start;
+    align-content: flex-start;
+  }
+
+  .sections__section {
+    display: flex;
+    flex-flow: column wrap;
+    max-width: 768px;
+    height: 100%;
+    justify-content: flex-start;
+    align-content: center;
+    background: url(https://i.ibb.co/YBt2P1f/Tesla-Wordmark-Red-ipad.png) no-repeat;
+    background-size: 100%;
+    background-position: top 90px center;
+    margin: 0;
+  }
+
+  .section__greeting {
+    display: flex;
+    width: 251px;
+    height: 19px;
+    font-size: 15px;
+    text-align: center;
+    margin: -14px auto 87px auto;
+    padding: 0;
+  }
+
+  .section__image-car {
+    display: flex;
+    width: 672.6px;
+    height: 327.64px;
+    padding: 0;
+    background: url(https://i.ibb.co/DWqCb1v/tesla-png-tesla-png-hd-1636-ipad.png);
+    background-size: contain;
+    margin-bottom: 0 0 36.55px 0;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5851407)